wifiphisher/phishing-pages/oauth-login/upgrading.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
Instead of proceeding to the next OAuth step, I believe it makes more sense to provide a simple error message here. Something like "Oops, an error occured! Our engineers were notified. Please be patient as we are working on it."